pom.xml

```diff
@@ -125,52 +125,19 @@
 
     <profiles>
         <profile>
-            <id>re-build</id>
-            <activation>
-                <property>
-                    <name>license.url</name>
-                </property>
-            </activation>
+            <id>checkstyle</id>
```

Comment:
if we are going to have the Travis build run checkstyle and fail if there are errors, I would prefer to just have this run by default -- I know I will never remember to run the separate checkstyle profile before making each PR

Reply:
Well I wanted to include it to regular build at first, but I wouldn't do that by default. Mainly because of when you are coding something and testing some function, you don't want to be bothered about some checkstyle stuff when most of it is there just temporarily. That is why I think that making it in separate profile is better way. And in case you forget to check it by yourself, there is that checkstyle option in pipeline :-) Do you still think that this is not the way to go?

Reply:
I would rename this profile to "troll" :)

Reply:
yea I guess it's OK to have it be a separate profile for now. If it ends up being a problem we can always adjust it in the future